Simple Resource Bundler issue

Howdy!
I'm doing fun things with embedded things - tl;dr I'm having issues with Simple resource bundler

I added a dev-dependencies on it, here's my `package.json`

```json
"devDependencies": {
    "aws-cdk-assert": "^0.6.0",
    "aws-cdk-toolkit": "^0.6.0",
    "aws-sdk": "^2.135.0",
    "pkglint": "^0.6.0",
    "simple-resource-bundler": "^0.6.0"
  },
  "dependencies": {
    "aws-cdk": "^0.6.0",
    "aws-cdk-iam": "^0.6.0",
    "aws-cdk-s3": "^0.6.0",
    "aws-cdk-resources": "^0.6.0",
    "aws-cdk-ec2":"^0.6.0",
    "aws-cdk-custom-resources":"^0.6.0",
    "aws-cdk-lambda":"^0.6.0",
    "aws-cdk-route53":"^0.6.0",
    "aws-cdk-acm":"^0.6.0",
    "aws-cdk-cloudfront": "^0.6.0"
  }
```

And changed my `scripts` to

```json
"scripts": {
    "prepare": "simple-resource-bundler -o lib && jsii && tslint -p .",
    "watch": "tsc -w",
    "lint": "tsc && tslint -p . --force",
    "test": "nodeunit test/test.*.js"
  },
```

but that gets angry with:

```
module.js:559
    throw err;
    ^

Error: Cannot find module 'source-map-support/register'
    at Function.Module._resolveFilename (module.js:557:15)
    at Function.Module._load (module.js:484:25)
    at Module.require (module.js:606:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/brelandm/workplace/bones_constructs/src/BONESConstructs/node_modules/simple-resource-bundler/bundler.js:41:1)
    at Module._compile (module.js:662:30)
    at Object.Module._extensions..js (module.js:673:10)
    at Module.load (module.js:575:32)
    at tryModuleLoad (module.js:515:12)
    at Function.Module._load (module.js:507:3)
```

Which seems to indicate it needs `https://www.npmjs.com/package/@types/source-map-support` maybe?

I'm not sure - but I think we can also remove it?

(I commented it out in my `node_modules` and it at least pretends to execute)

--- a/packages/simple-resource-bundler/package.json
+++ b/packages/simple-resource-bundler/package.json
@@ -25,7 +25,8 @@
   },
   "dependencies": {
     "fs-extra": "^4.0.2",
-    "yargs": "^9.0.1"
+    "yargs": "^9.0.1",
+    "source-map-support": "^0.5.6"
   },
   "repository": {
     "url": "git://github.com/awslabs/aws-cdk",

